modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -13,7 +13,7 @@ Running cluster monitoring with persistent storage means that your metrics are s
 
 * Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods.
 
-* Make sure you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The PVs should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
+* Verify that you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has two replicas, you need four PVs to support the entire monitoring stack. The PVs are available from the Local Storage Operator, but not if you have enabled dynamically provisioned storage.
```

Comment:
Because Prometheus has two replicas and Alertmanager has two replicas
->
Because Prometheus and Alertmanager both have two replicas

Reply:
changed per your suggestion